### ISSUE TITLE
bazel build command

### ISSUE THREAD

**Issue by YasineNifa** [mozilla/DeepSpeech#2123]
Hey !!
I am following this tutorial : https://github.com/mozilla/DeepSpeech/tree/master/native_client 
When I do this command : 
`bazel build --config=monolithic -c opt --copt=-O3 --copt="-D_GLIBCXX_USE_CXX11_ABI=0" --copt=-fvisibility=hidden //native_client:libdeepspeech.so //native_client:generate_trie`
I get this error : 
`ERROR: Skipping '//native_client:libdeepspeech.so': no such package 'native_client': BUILD file not found on package path`
It seems that it does sont find the libdeepspeech.so. Is it a problem with the version ?
How can I solve it

**Comment by lissyx** [comment]
@YasineNifa Why do you keep posting questions on Github when we document Discourse ? Have you properly read the documentation and checked out our TensorFlow repo and branches ?

**Comment by lissyx** [comment]
@YasineNifa https://github.com/mozilla/DeepSpeech/tree/master/native_client#tensorflow-clone--checkout

**Comment by lissyx** [comment]
https://github.com/mozilla/DeepSpeech/tree/master/native_client#compile-libdeepspeechso--generate_trie

**Comment by lissyx** [comment]
@YasineNifa And why do you need to rebuild ?

**Comment by YasineNifa** [comment]
Hey !
Because I ask there and I think the publication is not yet approved.

**Comment by lissyx** [comment]
>  Because I ask there and I think the publication is not yet approved.

Can you elaborate ? Are you having issues logging in on Discoure ? Posting ?

**Comment by YasineNifa** [comment]
I already create an account on Discoure, but I think I have a problem with posting
Thx for your time @lissyx

**Comment by lissyx** [comment]
> I already create an account on Discoure, but I think I have a problem with posting

Well, you should have told that earlier, we need to unblock that. Please share more details on your issue on Discourse, there's nothing I can do without more informations.

**Comment by YasineNifa** [comment]
unfortunately I can not post on Discourse 
Here is a message I receive : 
```
Hello,

This is an automated message from Mozilla Discourse to let you know that your account has been temporarily placed on hold as a precautionary measure.

Please do continue to browse, but you won’t be able to reply or create topics until a staff member reviews your most recent posts. We apologize for the inconvenience.

For additional guidance, refer to our community guidelines.



```

**Comment by hmitsch** [comment]
Looping in @LeoMcA for Discourse help.

**Comment by hmitsch** [comment]
Hi @YasineNifa, the Discourse moderation tools put you "on hold". I have now changed that. Could you please try to post on Discourse again. Sorry for the inconveniences.

Best regards,
   Henrik

**Comment by YasineNifa** [comment]
Hey @hmitsch 
Cool now I can post on Discourse 
Thx for the help :)
